### ISSUE TITLE
How to Center Allign the Text ?

### ISSUE THREAD

**Issue by Adnan-addi** [ViHtarb/Tooltip#33]
i have large text in this tooltip which is left aligned by default , i want to make it center , 

Using set gravity changes tooltip location...

**Comment by ViHtarb** [comment]
yes, in next SNAPSHOT version i implement this feature, w8 please)
